app/components/Header.tsx

```diff
@@ -132,6 +133,15 @@ export function Header() {
                       >
                         Peer Endorsements
                       </Link>,
+                      !idp && (
+                        <Link
+                          key="password"
+                          to="/user/password"
+                          onClick={() => setUserMenuIsOpen(!userMenuIsOpen)}
```

Comment:
Rebase from the main branch. This should now be:

```suggestion
                          onClick={onClickUserMenuItem}
```